vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -338,6 +344,70 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
         return pathResult;
     });
 
+    const dotnetUninstallPublicRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.uninstallPublic}`, async () =>
+    {
+        const existingInstalls = await InstallTrackerSingleton.getInstance(globalEventStream, vsCodeContext.globalState).getExistingInstalls(true);
+
+        const menuItems = existingInstalls?.sort(
+        function(x : InstallRecord, y : InstallRecord) : number
+        {
+            if(x.dotnetInstall.installMode === y.dotnetInstall.installMode)
+            {
+                return x.dotnetInstall.version.localeCompare(y.dotnetInstall.version);
+            }
+            return x.dotnetInstall.installMode.localeCompare(y.dotnetInstall.installMode);
+        })?.map(install =>
+        {
+            return {
+                label : `.NET ${(install.dotnetInstall.installMode).toUpperCase()} ${install.dotnetInstall.version}`,

```

Comment:
Since installMode has only 3 values, use .NET styling rather than upper case.

Reply:
Thank you, I think this is a good idea.